### PR TITLE
wording fix

### PR DESCRIPTION
**Pull request by verygenericname**

### CODE REVIEW

**Comment by Cryptiiiic** [comment]
Any other typos?

**Comment by verygenericname** [comment]
that's all I saw for now

**Comment by verygenericname** [comment]
that's all i could find

File: futurerestore/futurerestore.cpp [hunk 1]
```diff
@@ -1198,7 +1198,7 @@ void futurerestore::doRestore(const char *ipsw){
 
         /* now we load the iBEC */
         retassure(!recovery_send_ibec(client, build_identity),"ERROR: Unable to send iBEC\n");
-
+
```
Comment (ghost) [inline futurerestore/futurerestore.cpp]: </img>

File: futurerestore/futurerestore.cpp [hunk 2]
```diff
@@ -890,22 +890,22 @@ void futurerestore::doRestore(const char *ipsw){
...
+        error("ECID inside APTicket does not match the device's ECID\n");
         printf("APTicket is valid for %16llu (dec) but device is %16llu (dec)\n",im4mEcid,deviceEcid);
         if(_skipBlob) {
             info("[WARNING] NOT VALIDATING SHSH BLOBS ECID!\n");
         } else {
             reterror("APTicket can't be used for restoring this device\n");
         }
     }else
-        printf("Verified ECID in APTicket matches device ECID\n");
+        printf("Verified ECID in APTicket matches the device ECID\n");
```
Comment (m1stadev) [inline futurerestore/futurerestore.cpp]:
```suggestion
        printf("Verified ECID in APTicket matches the device's ECID\n");

File: futurerestore/futurerestore.cpp [hunk 3]
```diff
@@ -890,22 +890,22 @@ void futurerestore::doRestore(const char *ipsw){
...
 
     if (_client->image4supported) {
-        printf("checking APTicket to be valid for this restore...\n");
+        printf("checking if the APTicket is valid for this restore...\n");
         uint64_t deviceEcid = getDeviceEcid();
 
         if (im4mEcid != deviceEcid) {
-            error("ECID inside APTicket does not match device ECID\n");
+            error("ECID inside APTicket doesn't match device ECID\n");
```
Comment (m1stadev) [inline futurerestore/futurerestore.cpp]:
```suggestion
            error("ECID inside APTicket doesn't match the device's ECID\n");

**Review by m1stadev — CHANGES_REQUESTED** [review]
consistency

**Review by m1stadev — APPROVED** [review]

**Comment by Cryptiiiic** [comment]
@verygenericname fix conflicts

**Comment by verygenericname** [comment]
laziness is fun, i'm just gonna make a new pr cuz why not

**Comment by Cryptiiiic** [comment]
please add the spacing back and the end file newline

**Comment by Cryptiiiic** [comment]
bump

**Pull request closed without merging**